### PR TITLE
use cattle system registry

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

**Review by brendarearden — APPROVED** [review]
LGTM

**Comment by maggieliu** [comment]
related to: https://github.com/rancher/rancher/issues/28741

File: packages/rancher-logging/rancher-logging.patch [hunk 1]
```diff
@@ -87,4 +87,6 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +    tag: 0.1.0
 +
 +global:
-+  systemDefaultRegistry: ""
++  cattle:
++    systemDefaultRegistry: ""
```
Comment (rmweir) [inline packages/rancher-logging/rancher-logging.patch:91]: nit: add new line

**Review by rmweir — CHANGES_REQUESTED** [review]

**Review by rmweir — APPROVED** [review]
LGTM with 1 nit

File: packages/rancher-logging/rancher-logging.patch (hunk 1 above)
Comment (paynejacob) [reply packages/rancher-logging/rancher-logging.patch:91]: This will more than likely collide with another pr, I will fix it in there.

**Review by cbron — APPROVED** [review]

**Pull request merged**